phoenix-core/src/it/java/org/apache/phoenix/end2end/UnionAllIT.java

```diff
@@ -1065,4 +1065,44 @@ public void testCoerceExpr() throws Exception {
             conn.close();
         }
     }
+
+    @Test
+    public void testUnionAllWithLiteralColumn() throws Exception {
+        String tableName1 = generateUniqueName();
+        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        Connection conn = DriverManager.getConnection(getUrl(), props);
+        conn.setAutoCommit(false);
+
+        try {
```

Comment:
Could you please use try-with-resources?

Reply:
Done, thanks.